extensions/ql-vscode/package.json

```diff
@@ -647,15 +647,15 @@
         "enablement": "codeql.hasQLSource"
       },
       {
-        "command": "codeQL.mockGitHubApiServer.startRecording",
+        "command": "codeQLMockGitHubApiServer.startRecording",
```

Comment:
I'm not sure I agree with this approach.. I think we still want all the commands provided by our extension to be prefixed with `CodeQL: `. Why have we gone against it?

Reply:
I'd rather we do that if that's okay. Then we know that everything starts with `CodeQL:`.